### ISSUE TITLE
Feature suggestion: auto-update

### ISSUE THREAD

**Issue by mcrampon** [hdlx/AltAppSwitcher#14]
Would be super nice if the app could automatically check for updates on start up, and download them😄

**Comment by hdlx** [comment]
I agree 🙂 A first intermediate step would be notifying the user without auto download.  And "Do not search for update" checkbox would update the options accordingly.

**Comment by hdlx** [comment]
Done since a few versions already :-)
